editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -187,7 +190,8 @@ impl Fsm for PathToolFsmState {
 						let include_handles = tool_data.shape_editor.selected_layers_ref();
 						tool_data.snap_manager.add_all_document_handles(document, input, &include_handles, &[], &selected_points.points);
 
-						tool_data.drag_start_pos = input.mouse.position - selected_points.offset;
+						tool_data.drag_start_pos = input.mouse.position;
+						tool_data.previous_mouse_position = input.mouse.position - selected_points.offset;
```

Comment:
Is `previous_mouse_position` an accurate and descriptive name for this? I'm having trouble seeing how the current mouse position minus the selected points offset is the "previous mouse position". To me, that implies the position where the mouse was located at the moment the drag began-- except that's what the previous line does when it sets `drag_start_pos`. Is there a better name for `previous_mouse_position`?

Reply:
It is the mouse position subtract any snapping offset (which is usually nothing) - so I would say `previous_mouse_position` is a reasonable name.